### ISSUE TITLE
Adapt views to recognize agent requests

### ISSUE THREAD

**Issue by samircury** [dmwm/WMCore#3672]
I see views that filter like :
{{{
  if (doc.type == "reqmgr_request"){
    var summary = {};
}}}
We also provide similar documents from the agent, but with a different tag :

{{{
{
   "_id": "62802086d056695706cd50e96c1d3547",
   "_rev": "1-425ed166a5eb923fd8e76ba672f0810e",
   "status": {
       "submitted": {
           "running": 7
       }
   },
   "agent_team": "team1,team2,cmsdataops",
   "workflow": "Express_Run191691_StreamExpress",
   "timestamp": 1335951189,
   "sites": {
       "T0_CH_CERN": {
           "submitted": {
               "running": 7
           }
       }
   },
   "agent": "WMAgentCommissioning",
   "agent_url": "vocms114.cern.ch",
   "type": "agent_request"
}
}}}

So basically we either ask the view to look for both request types (if they are compatible), or select a different behaviour for each kind of request. I will test it and report back here.

**Comment by amaltaro** [comment]
@ticoann do we still need to fix it? otherwise we should close since it's 3 years old.
